### ISSUE TITLE
Can GTX 965m train the voc 2007 model?

### ISSUE THREAD

**Issue by chunji96** [balancap/SSD-Tensorflow#131]
I try ro use gtx365m to train voc2007,but it show that memory is not enough,but when i change rhe batch_size=8,it can work, while,the loss =20 at step 120.
### can gtx 965m be enough to use? Or what parameter shou i change?
